src/components/ec-modal/ec-modal.vue

```diff
@@ -222,151 +225,135 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/settings/colors/index';
-@import '../../scss/tools/index';
-
-$ec-modal-bg: rgba($level-2-doc-bodies-bg, 0.2) !default;
-$ec-modal-color: $level-3-body-and-headings !default;
-$ec-modal-content-bg: $white !default;
-$ec-modal-content-footer-bg: $level-7-backgrounds !default;
-$ec-modal-close-btn-fill: $level-4-interactive-elements !default;
-$ec-modal-close-btn-fill-hover: $level-4-tech-blue !default;
+<style>
+@import '../../styles/tools/scrollbars.css';
+@import '../../styles/tools/transitions.css';
 
 .ec-modal {
-  background: $ec-modal-bg;
-  width: 100vw;
-  height: 100vh;
-  position: fixed;
-  top: 0;
-  left: 0;
-  color: $ec-modal-color;
+  background: hsla(var(--ec-gray-color-level-2), 0.2);
 
-  @include z-index-modal;
+  @apply tw-w-screen;
+  @apply tw-h-screen;
+  @apply tw-fixed;
+  @apply tw-inset-0;
+  @apply tw-text-gray-3;
+  @apply tw-z-modal;
 
   &__content {
-    width: calc(100% - 24px);
-    max-width: 680px;
-    position: relative;
-    top: 50%;
-    left: 50%;
-    background: $ec-modal-content-bg;
+    width: calc(100% - theme('spacing.24'));
+    max-width: var(--ec-modal-sm-max-width);
+    max-height: calc(100% - theme('spacing.24'));
     transform: translate(-50%, -50%);
```

Comment:
can we use this? `@apply tw--translate-x-1/2 tw--translate-y-1/2`I'm curious

Reply:
Not really :/ Tried it. this will apply twice the transform property and will overwrite the first value with the second.